api/src/certification/shared/domain/models/Frameworks.js

```diff
@@ -1,13 +1,34 @@
+import { NotFoundError } from '../../../../shared/domain/errors.js';
+
 /**
  * Frameworks scopes
  * @readonly
  * @enum {string}
  */
-export const Frameworks = Object.freeze({
+export const FrameworksEnum = Object.freeze({
```

Comment:
```suggestion
export const FRAMEWORKS = Object.freeze({
```
Dans quelle mesure utilise-t-on les majuscules pour définir les enums (comme pour `ABORT_REASONS`) ?
Pratique non généralisée ?

Reply:
Done